client/my-sites/site-settings/settings-reading/main.tsx

```diff
@@ -1,14 +1,56 @@
 import config from '@automattic/calypso-config';
 import { useTranslate } from 'i18n-calypso';
+import { pick } from 'lodash';
 import DocumentHead from 'calypso/components/data/document-head';
 import FormattedHeader from 'calypso/components/formatted-header';
 import Main from 'calypso/components/main';
+import { SiteSettingsSection } from '../reading-site-settings';
+import wrapSettingsForm from '../wrap-settings-form';
 import { NewsletterSettingsSection } from './newsletter-settings-section';
 import { RssFeedSettingsSection } from './rss-feed-settings-section';
-import { SiteSettingsSection } from './site-settings-section';
 
 const isEnabled = config.isEnabled( 'settings/modernize-reading-settings' );
 
+// Settings are not typed yet, so we need to use `unknown` for now.
+type Settings = unknown;
+
+const getFormSettings = ( settings: Settings ) => pick( settings, [ 'posts_per_page' ] );
+
+type Fields = {
+	posts_per_page?: number;
+};
+
+type ReadingSettingsFormProps = {
+	fields: Fields;
+	onChangeField: ( field: string ) => ( event: React.ChangeEvent< HTMLInputElement > ) => void;
+	handleSubmitForm: ( event: React.FormEvent< HTMLFormElement > ) => void;
+	isRequestingSettings: boolean;
+	isSavingSettings: boolean;
+};
+
+const ReadingSettingsForm = wrapSettingsForm( getFormSettings )(
+	( {
+		fields,
+		onChangeField,
+		handleSubmitForm,
+		isRequestingSettings,
+		isSavingSettings,
+	}: ReadingSettingsFormProps ) => {
+		return (
+			<form onSubmit={ handleSubmitForm }>
```

Comment:
Other settings sections forms set the `id` and `className` in the following way:

```suggestion
			<form id="site-settings" onSubmit={ handleSubmitForm } className="site-settings__reading-settings">
```

Reply:
Thanks for highlighting this. I did not include the id and classes on purpose but I have not truly verified that those are not needed. I will have a look at this now.